src/overlays/actors/ovl_Bg_Po_Syokudai/z_bg_po_syokudai.c

```diff
@@ -9,7 +15,26 @@ void BgPoSyokudai_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void BgPoSyokudai_Update(Actor* thisx, GlobalContext* globalCtx);
 void BgPoSyokudai_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-/*
+static ColliderCylinderInit sCylinderInit = {
+    { COLTYPE_METAL_SHIELD, 0x00, 0x0D, 0x39, 0x20, COLSHAPE_CYLINDER },
+    { 0x00, { 0x00000000, 0x00, 0x00 }, { 0xFFCFFFFF, 0x00, 0x00 }, 0x00, 0x01, 0x01 },
+    { 12, 60, 0, { 0, 0, 0 } },
+};
+
+Color_RGBA8 primColors[] = {
+    { 255, 170, 255, 255 },
+    { 255, 200, 0, 255 },
+    { 0, 170, 255, 255 },
+    { 170, 255, 0, 255 },
+};
+
+Color_RGBA8 envColors[] = {
```

Comment:
primColors and envColors are very generic names so it would be a good idea to declare them with `static` so they can't ever conflict with anything outside the file